packages/hui-pages/src/routes/IconDocs/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { Grid, Row } from 'react-flexbox-grid'
+
+import Icon from '@hixme-ui/icon'
+// import Table from '@hixme-ui/table'
+// import Title from '@hixme-ui/title'
+
+import { Code, DocsTitle } from '../../components'
+
+const IconDocs = () => (
+  <div>
+    <DocsTitle
+      title='Icon'
+      subtitle='A Hixme UI styled icon component'
+    />
+    <Grid fluid>
+      <Row center='xs' style={{ marginBottom: '30px' }}>
+        <Icon menu size={3} primary />
```

Comment:
This is the same problem I was having. A primary menu icon is what we're trying to render, but a black one is what actually get renders on the page -- and black is the color that is specified as the default value on the getTextColor call.

From `/packages/icon/src/Icon.js`:
```jsx
<UseIcon
  color={color || getTextColor(props, 'black')}
  fill={color || getTextColor(props, 'black')}
  style={{ maxHeight: getFontSize(props) }}
/>
```

It's always falling through to that default value, regardless of the color props we supply. It breaks when removed.